### ISSUE TITLE
Update cycle dependencies

### ISSUE THREAD

**Issue by wclr** [cyclejs/cycle-time-travel#28]
outdated a little bit, cycle deps versions and the project itself, goting to contitue?

**Comment by Widdershin** [comment]
Yep, currently working on https://github.com/Widdershin/cycle-restart which will allow me to make cycle-time-travel much better, and much easier to use. Expect to see major changes in the next month or so.
